app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreen.kt

```diff
@@ -0,0 +1,50 @@
+package org.greenstand.android.TreeTracker.splash
+
+import android.widget.Toast
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.NavController
+import kotlinx.coroutines.delay
+import org.greenstand.android.TreeTracker.BuildConfig
+import org.greenstand.android.TreeTracker.R
+import org.greenstand.android.TreeTracker.preferences.PreferencesMigrator
+import org.greenstand.android.TreeTracker.view.TreeTrackerTheme
+import org.koin.core.context.GlobalContext
+import timber.log.Timber
+
+
+@Composable
+fun SplashScreen(preferencesMigrator: PreferencesMigrator, navController: NavController?) {
+    LaunchedEffect(true) {
+        Timber.tag("BuildVariant").d("build variant: ${BuildConfig.BUILD_TYPE}")
+        preferencesMigrator.migrateIfNeeded()
+        delay(100)
```

Comment:
Question: What does this delay do? I've never seen this before. Curious on what it does!

Reply:
It's just to show the splash screen for 1 second, but it's .1 second while building out the app

Reply:
It pauses the coroutine.